### PR TITLE
Add getter for current index

### PR DESCRIPTION
**Pull request by vadymmarkov**
Sometimes it's useful to get the index of the current page.

### CODE REVIEW

File: Source/PagesController.swift [hunk 1]
```diff
@@ -13,7 +13,7 @@ import UIKit
 
   lazy var pages = Array<UIViewController>()
 
-  var currentIndex: Int = 0
+  public private(set) var currentIndex: Int = 0
```
Comment (kostiakoval) [inline Source/PagesController.swift]: `public private(set) var currentIndex = 0`
I would remove types when possible :)

**Comment by zenangst** [comment]
@vadymmarkov :cake:

**Comment by kostiakoval** [comment]
My suggestion was approved :) 
Yeaaaa :tada: 
Giphy

**Comment by zenangst** [comment]
@kostiakoval 
chuck_norris_salute

**Pull request merged**